mlflow/R/mlflow/R/tracking-runs.R

```diff
@@ -479,7 +480,7 @@ mlflow_log_artifact <- function(path, artifact_path = NULL, run_id = NULL, clien
              client = client
   )
 
-  mlflow_list_artifacts(run_id = run_id, path = artifact_path, client = client)
+  invisible(mlflow_list_artifacts(run_id = run_id, path = artifact_path, client = client))
```

Comment:
TODO: I wouldn't expect this to be a breaking change, but should confirm.

Reply:
Why do you want to make this invisible?

Reply:
Mainly just because it seems weird to print every time we log an artifact (this could generate a lot of output if users log many artifacts - e.g. we use `invisible` in `mlflow_log_param` and `mlflow_log_metric` for similar reasons), although fine not changing it as well